test/framework/verify/pod.go

```diff
@@ -155,6 +155,15 @@ func (v *PodVerification) WindowsPodHaveIPv4Address(pod *v1.Pod) {
 
 }
 
+func (v *PodVerification) WindowsPodHaveIPv4AddressFromPrefix(pod *v1.Pod) {
+	By("matching the prefix-deconstructed IPv4 from annotation to the pod IP")
+	ipAddWithCidr, found := pod.Annotations["vpc.amazonaws.com/PrivateIPv4Address"]
+	Expect(found).To(BeTrue())
+	// Remove the CIDR and compare pod IP with the IP annotated from VPC Controller
+	Expect(pod.Status.PodIP).To(Equal(strings.Split(ipAddWithCidr, "/")[0]))
```

Comment:
nit: I recall you added a func which can help extract IP and mask from the string? Can we use that here if I recall correctly?

Reply:
Yea there is a helper function to deconstruct prefix into IPs `DeconstructIPsFromPrefix()`. Here we will need to check if the deconstructed IPs contain the pod's IP. Should I submit a new PR for the improvement or add commit to this PR?

Reply:
Adding a commit is good enough. Thanks.

Reply:
We can merge this and if you have another PR coming. We can update there. Thanks.